src/styles/blocks/animation.scss

```
@@ -0,0 +1,8 @@
+@keyframes move {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
```

Add the empty line to the end of all files
```suggestion
}

```